eqllib/analytics/execution/T1047-suspicious-image-loads-via-commonly-abused-executable.toml

```diff
@@ -0,0 +1,20 @@
+[analytic.metadata]
+categories = ["detect"]
+confidence = "low"
+contributors = ["Daniel Stepanic"]
+created_date = "12/04/2019"
+description = "Identifies suspicious image load events tied to Koadic stager processes."
+id = "45835773-d859-4d44-8ef6-5d95f7538612"
+name = "Suspicious Image Loads via Commonly Abused Executables"
+os = ["windows"]
+tactics = ["Execution"]
+tags = ["koadic"]
+techniques = ["T1047","T1064"]
+updated_date = "12/04/2019"
+
+[analytic]
+query = '''
+sequence by unique_pid
+  [process where subtype.create and process_name in ("mshta.exe", "rundll32.exe")]
+  [image_load where image_name in ("jscript9.dll", "msxml6.dll", "npmproxy.dll", "winhttp.dll", "wmiutils.dll")]
```

Comment:
I think it's actually normal for mshta.exe to load most of these.

Reply:
Yeh, I guess we could remove `jscript9.dll`.  I kinda like the others, they all show a sign of dipping into other functionality.

Reply:
Removed two of the DLL's for now.